config/software/delivery-cli.rb

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright 2015 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name "delivery-cli"
+default_version "master"
+
+source git: "https://github.com/chef/delivery-cli.git"
+
+dependency "openssl"
+dependency "rust"
```

Comment:
Where does this end up in the build order? I'd like rust to be as early as possible so we cache it more often.